org.netsurf_browser.NetSurf.yaml

```diff
@@ -0,0 +1,68 @@
+---
+app-id: org.netsurf_browser.NetSurf
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+command: netsurf-gtk3
+finish-args:
+  - "--filesystem=xdg-download"
+  - "--share=ipc"
+  - "--share=network"
+  - "--socket=x11"
+  - "--socket=fallback-x11"
```

Comment:
It's not clear from documentation but it's one or the other, not both. Just `fallback-x11` is enough if it works natively on Wayland.

Reply:
Thankyou, I have made that change

Reply:
@barthalion Are you sure? I thought older versions of Flatpak that pre-date fallback-x11 also need x11 in order to have any DISPLAY set. https://github.com/flatpak/flatpak/blob/master/doc/flatpak-build-finish.xml#L139-L144

Reply:
Even Ubuntu 18.04 ships a version with fallback-x11. We can't realistically care about so much backwards compatibility forever.